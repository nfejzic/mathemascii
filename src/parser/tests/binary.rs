use super::Snapshot;

super::test_snap!(frac, "frac 1.44 2 frac(x)(3.14)");
