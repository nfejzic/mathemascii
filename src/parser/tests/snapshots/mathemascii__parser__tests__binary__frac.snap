---
source: src/parser/tests/binary.rs
expression: "Snapshot((input, math))"
---
frac 1.44 2 frac(x)(3.14)

Expression {
| Fraction(
| | Number("1.44"),
| | Number("2")
| )
}

Expression {
| Fraction(
| | OpenParen
| | | Expression {
| | | | Variable("x")
| | | }
| | |
| | CloseParen,
| | OpenParen
| | | Expression {
| | | | Number("3.14")
| | | }
| | |
| | CloseParen
| )
}


